### ISSUE TITLE
TS|TSX  files are linted with useTypescriptIncrementalApi

### ISSUE THREAD

**Issue by theoomoregbee** [TypeStrong/fork-ts-checker-webpack-plugin#396]
My current config: 

```js
 new ForkTsCheckerWebpackPlugin({
  // useTypescriptIncrementalApi: false,
      async: false,
      eslint: true,
      ignoreLintWarnings: false,
    }),
```

created the variable without using it anywhere else
```js
const me = 'something'
```

#### Actual output
```
[cI1f] /Users/theophy/dev/..../../Component.tsx 2.17 KiB {Page} [built]
+ 13 hidden modules
```

#### Expected output 

```sh
[cI1f] /Users/theophy/dev/..../../Component.tsx 2.17 KiB {Page} [built]
+ 13 hidden modules
WARNING in /Users/theophy/dev/..../../Component.tsx
WARNING in /Users/theophy/dev/..../../Component.tsx(20,9):
@typescript-eslint/no-unused-vars: 'me' is assigned a value but never used.
``` 

Changing `useTypescriptIncrementalApi` to `false` works but is much slower.

related to #219

**Comment by piotr-oles** [comment]
@theoomoregbee
Please try [fork-ts-checker-webpack-plugin@alpha](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/tree/alpha) - I've published a new version which should resolve this issue 🚀
I will close this issue to clean-up the backlog. If this release didn't solve the issue, we can re-open this :)
